packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -164,6 +164,8 @@ class CallContext {
       /* @conditional-compile-remove(DNS) */
       deepNoiseSuppressionOptions?: {
         onResolveDependency?: () => Promise<DeepNoiseSuppressionEffectDependency>;
+        deepNoiseSuppressionOnByDefault?: boolean;
```

Comment:
are we including both of these options because Contoso can turn it on by default and not give the user the option to turn it off? Do we have a way for the user to recover here if there is an issue DNS that stops them from being heard at all? I don't know if this is something that can impact us but I have seen this happen on applications like Discord.

Reply:
Yeah, the idea is if Contoso plans to enforce DNS in the call by default and doesn't want to give option to users to turn it off/on. I haven't thought about the scenario of user losing the voice, let me check with calling SDK and we can do some error handling to let contoso know